### ISSUE TITLE
[FEATURE] Adding proper meta tags to the website

### ISSUE THREAD

**Issue by Avinash905** [tamalCodes/Milan#855]
### What would you like to share?

**Issue:** The website does not have meta tags like title, keywords, language, etc.

**Impact:** This means that the website is not as visible in search results, and users are less likely to click on the website's links.

**Resolution:** The website should contain important meta tags. This will help the website rank higher in search results and attract more visitors.

### 🥦 Browser

Google Chrome

### Checklist ✅

- [X] I checked and didn't find similar issue
- [X] I have read the [Contributing Guidelines](https://github.com/tamalCodes/Milan/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (blank for no)

**Comment by tamalCodes** [comment]
Great issue though we tried our best to optimize the site's meta tags - we used a 3rd party library called [React-Helmet](https://github.com/nfl/react-helmet#readme), a lot of it is "really" old.  Assigning this to you ⚡

### Good PR makes a Good Developer !

We at Milan work hard to maintain the structure, and use conventional Pull request titles and commits.
**Please make sure that you have read our documentation on [Commiting guidelines](https://github.com/IAmTamal/Milan/blob/main/CONTRIBUTING.md#commits-) and [Pull request guidelines](https://github.com/IAmTamal/Milan/blob/main/CONTRIBUTING.md#pull-requests-).**

Without a proper template for the PR, not following the guidelines and spam might get the pull request closed, or banned.
